README.md

```diff
@@ -2,19 +2,48 @@
 <a href="https://autorelease.general.dmz.palantir.tech/palantir/javapoet"><img src="https://img.shields.io/badge/Perform%20an-Autorelease-success.svg" alt="Autorelease"></a>
 </p>
 
-JavaPoet
-========
+<p align="center">
+<a href="https://search.maven.org/artifact/com.palantir.javapoet/javapoet"><img src="https://img.shields.io/maven-central/v/com.palantir.javapoet/javapoet" alt="Maven Central Version"></a>
+<a href="https://javadoc.io/doc/com.palantir.javapoet/javapoet"><img src="https://javadoc.io/badge2/com.palantir.javapoet/javapoet/javadoc.svg" alt="Javadoc Version"></a>
```

Comment:
Not completely sure how useful the javadoc.io badge is. It seems large parts of the API are not documented (also a problem of the upstream project). But maybe it is useful nonetheless to at least see the publicly available types and methods.

Reply:
I don't have a strong opinion on this one :-)